_apps/mason.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Mason
+layout: app
+cite: fu_mi_publications962
+contact: Manuel Holtgrewe
+category: official
+links:
+  download: http://packages.seqan.de/mason/
+  source: https://github.com/seqan/seqan/tree/master/apps/mason2
+redirect_from:
+  - /projects/mason/
```

Comment:
How did you find these old links again?

Reply:
here: https://web.archive.org/web/20141104072722/http://www.seqan.de/projects/mason/